docs/proposals/VIP_TEMPLATE.md

```diff
@@ -0,0 +1,135 @@
+---
+layout: default
+title: Venice Improvement Plan
+parent: Developer Guides
```

Comment:
Hmm, this seems inconsistent with the path below. It might work but may make the page structure confusing. Either we have a new top-level section for proposals, at the same level as the dev guides section, or else proposals are nested under dev guides.

Reply:
So Proposals are at the same level as the dev guide but looks like the parent in these sections points to "Community Guides". Will do that as proposals are part of how we interact with the community. Looking into that one, I figured we also have design guidelines pointing to RFC wiki on the contributing page. Will correct that one too.

Reply:
Does this look alright @FelixGV